node-src/lib/getOptions.ts

```diff
@@ -148,7 +148,7 @@ export default function getOptions({
     log.setInteractive(false);
   }
 
-  if (!options.projectToken) {
+  if (!options.projectToken && !(options.projectId && options.userToken)) {
```

Comment:
Where does one get a user token to invoke the CLI in this new way?

Reply:
The Visual Tests addon does, using the OAuth flow.